### PR TITLE
[mlnx-ffb.sh] Update issu-version location

### PR DESCRIPTION
**Pull request by stepanblyschak**
#### Why I did it

ISSU version check fails due to inability to mount squashfs from 202211 on 201911

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

Put ISSU version file under platform directory

#### How to verify it

Warm-upgrade matrix:
- 201911 (with https://github.com/sonic-net/sonic-buildimage/pull/14928) to master
- 201911 (with https://github.com/sonic-net/sonic-buildimage/pull/14928) to 202211
- 202012 (with https://github.com/sonic-net/sonic-buildimage/pull/14927) to master
- 202205 (with this change cherry-picked) to master

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [x] 202106
- [x] 202111
- [x] 202205
- [x] 202211

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Review by volodymyrsamotiy — APPROVED** [review]

**Comment by vaibhavhd** [comment]
The change LGTM. However, the description says that the change has been tested for 201911 to 202211

Since this change is to master, I think the test should be done for all eligible base branches to master as target.

**Comment by vaibhavhd** [comment]
Why is this change not needed on 202205?

**Comment by liat-grozovik** [comment]
> Why is this change not needed on 202205?

it is needed this is why there is an ask to backport it.
there are PRs directly to 202012 and 201911 as they are not cleanly cherry picked

**Comment by stepanblyschak** [comment]
@vaibhavhd updated the test upgrade matrix in description

**Review by vaibhavhd — APPROVED** [review]

File: platform/mellanox/mlnx-ffb.sh [hunk 1]
```diff
@@ -49,41 +49,47 @@ check_sdk_upgrade()
         return "${FFB_SUCCESS}"
     fi
 
-    while :; do
-        mkdir -p "${FS_MOUNTPOINT}"
-        mount -t squashfs "${FS_PATH}" "${FS_MOUNTPOINT}" || {
-            >&2 echo "Failed to mount next SONiC image"
+    ISSU_VERSION_FILE_PATH="/etc/mlnx/issu-version"
```
Comment (qiluo-msft) [inline platform/mellanox/mlnx-ffb.sh:52]: >/etc/mlnx/issu-version [](http://example.com/codeflow?start=28&length=22)

Is this old path deprecated?

**Review by qiluo-msft — APPROVED** [review]

**Comment by vaibhavhd** [comment]
MSFT ADO: 24417508

**Comment by StormLiangMS** [comment]
@stepanblyschak @liat-grozovik cherry-pick failure, need separate PR to 202305.

**Pull request merged**